app/assets/stylesheets/layout_components/callouts/quote-callout.scss

```diff
@@ -1,7 +1,54 @@
 .quote-callout {
-  &__content {
-    @include absolute-center;
+  &__container {
+    display: flex;
+    background: $white;
+
+    @include breakpoint(medium up) {
+      flex-direction: row;
+    }
+
+    @include breakpoint(small only) {
+      flex-direction: column;
+      height: 600px; // HACK: IE 11 can't determine the flex container height for columns.
+    }
+  }
+
+  &__price {
+    background: $cerulean;
+    color: $white;
+    padding: $base-padding;
```

Comment:
I'm not sure we need the padding to be defined here and on line 28 below, I think we should be fine without it. What do you think?

Reply:
Done!